packages/mongo/oplog_tailing.js

```diff
@@ -148,23 +184,23 @@ _.extend(OplogHandle.prototype, {
     }
 
     if (self._stopped)
-      return;
+      return completeProcess();
```

Comment:
`return somethingWithNoReturnValue()` as a shorthand for
```
somethingWithNoReturnValue();
return;
```

is not common in the meteor codebase.

In any case, this can be done with try/finally instead of making sure to annotate all returns with a function call.